src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefReadonlyReturnTests.cs

```diff
@@ -52,7 +52,9 @@ class C
 
         return ref rs1;
     }
-}");
+}";
+
+            var comp = CompileAndVerify(source, parseOptions: TestOptions.Regular.WithPEVerifyCompatFeature());
```

Comment:
Is this something we're just keeping around until the verifier is updated?

Reply:
Yes. A last-resort workaround if compatibility with PEVerify is needed. - It does not always work with new features,as some features by definition do not conform with PEVerify, but at least for old features it can be used.

---
In reply to: [141783389](https://github.com/dotnet/roslyn/pull/22400#discussion_r141783389) [](ancestors = 141783389)